Luna/Luna/SettingsViewModel.swift

```diff
@@ -24,79 +24,82 @@ class SettingsViewModel
 		try authService.signOutUser()
 	}
     
-    func getUserData( completion: @escaping(_ error: Error?) -> Void )
+    
+    func createUserViewModel( completion: ( _ userViewModel: UserViewModel? ) -> Void )
```

Comment:
Can we just return `UserViewModel`? I could be wrong, but I don't see any asynchronous code in this call stack. Therefore, the completion block isn't needed. It's just acting like a return since it's running synchronously. It works, but whenever I see a completion block, it makes me believe that the method is running async.